### ISSUE TITLE
0.7.0 not on pub

### ISSUE THREAD

**Issue by yjbanov** [localvoid/uix#12]
Is this intentional?

**Comment by localvoid** [comment]
Yes, I wanted to find an easy way to add support for changing element properties, right now it works only with attributes.
